core/src/display_object/edit_text.rs

```diff
@@ -938,6 +952,18 @@ impl<'gc> EditText<'gc> {
         if let Some((text, _tf, font, params, color)) =
             lbox.as_renderable_text(edit_text.text_spans.displayed_text())
         {
+            // Whether the glyph should be scaled due to padding
+            // see: https://github.com/ruffle-rs/ruffle/issues/4051
+            context.transform_stack.push(&Transform {
+                matrix: Matrix::scale(
+                    1.0 - ((Self::INTERNAL_PADDING - edit_text.hscroll)
```

Comment:
I'm a little confused by this line. If I'm reading this correctly, the glyph gets more squished when the text box is scrolled to the right more?

Reply:
thanks！ I misunderstood the hscroll and scroll_offset fields. The reason I'm looking for auto-sized text is because the PADDING pixel translation is done when rendering the glyphs. Can I re-modify like below to do PADDING pixel scaling to glyphs. I tested some text and it works fine. I'm not sure I understand that correctly.
`            let glyph_scale = (1.0 - Self::INTERNAL_PADDING / edit_text.bounds.width().to_pixels() ) as f32;
            context.transform_stack.push(&Transform {
                matrix: Matrix::scale(glyph_scale, glyph_scale),
                ..Default::default()
            });`